publish.sh

```diff
@@ -0,0 +1,68 @@
+
+# Prerequisites
+if [ "$#" -ne 2 ]; then
+    echo "Illegal number of parameters. Plugin id and version should be passed."
+    exit 1
+fi
+
+PLUGIN_ID=$1
+PLUGIN_VERSION=$2
+
+
+# Login
+oc login ${CHE_OSO_CLUSTER} --insecure-skip-tls-verify=${CHE_OSO_TRUST_CERTS} --token=${CHE_OSO_USER_TOKEN} && oc project ${CHE_OSO_PROJECT}
```

Comment:
Out of curiosity: who is setting `${CHE_OSO_CLUSTER}`, `${CHE_OSO_CLUSTER}`, `${CHE_OSO_USER_TOKEN}` and ` ${CHE_OSO_PROJECT}` ? Are they always set in the Che dev machine?

Reply:
These env vars are used by oc login installer https://github.com/redhat-developer/rh-che/blob/master/plugins/fabric8-multi-tenant-manager/src/main/resources/installers/1.0.0/com.redhat.oc-login.script.sh#L15 and they make this script start registry in a user namespace, not a user-che namespace. In any case, che namespaces in tenants should be locked down, so we can't use che namespaces.
Yes, it is supposed to be injected in all the containers in a workspace